java/src/jmri/jmrix/openlcb/OlcbProgrammer.java

```diff
@@ -25,8 +25,8 @@
  * Depending on which type you have, only certain modes can be set. Valid modes
  * are specified by the class static constants.
  * <P>
- * You get a Programmer object from a {@link jmri.ProgrammerManager}, which in turn
- * can be located from the {@link jmri.InstanceManager}.
+ * You get a Programmer object from a ProgrammerManager, which in turn
```

Comment:
jmri.ProgrammerManager is showing as depreciated, so I figured that would prevent the invalid link when it does get removed. I suspect there is a correction for this, but a quick look around shows the depreciated link in may bits of code too.